src/components/TimerPlayer/TimerPlayer.js

```diff
@@ -63,20 +66,28 @@ const TimerPlayer = ({ id, timer, deleteTimer }) => {
       interval = setInterval(() => {
```

Comment:
Trying to remember how `setInterval`s and `setTimeout`s work in use effects. I'm fairly sure since you are changing the state inside, its causing another render, which reruns the `useEffect` again. So if you wanted use could use a singular `setTimeout` so you dont need to worry about managing the interval. That should help clean this `useEffect` up a little

Reply:
Actually since you are using the interval to do your counting, using a `setTimeout` might not be any better as you will still need to clear it when you pause etc